### PR TITLE
JArrayHelper => ArrayHelper for the frontend components

### PR DESCRIPTION
**Pull request by zero-24**
#### Summary of Changes

JArrayHelper => ArrayHelper
#### Testing Instructions

Please make sure that com_banners, com_config, com_content, com_tags & com_users (remind) still works as before.
Code Review ;)

### CODE REVIEW

File: components/com_content/models/articles.php [hunk 1]
```diff
@@ -427,7 +428,7 @@ protected function getListQuery()
 
 			if (!empty($first))
 			{
-				JArrayHelper::toString($authorAlias);
+				$authorAlias = ArrayHelper::toString($authorAlias);
 
 				foreach ($authorAlias as $key => $alias)
```
Comment (izharaazmi) [inline components/com_content/models/articles.php]: Foreach over string?

**Comment by grhcj** [comment]
I have tested this item :white_check_mark: successfully on 41cac17519a50c8aeefbf400b7aac795e055738d<br /><br />Comment by @izharaazmi isn't part of this pr imho.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10922">issues.joomla.org/joomla-cms/10922</a>.</sub>

**Comment by izharaazmi** [comment]
@grhcj Yes, It is! See https://github.com/joomla/joomla-cms/pull/8396 and https://github.com/joomla/joomla-cms/pull/8396#discussion_r44648607

Line 430 (old) was ineffective because `JArrayHelper::toString($authorAlias);` didn't change the `$authorAlias`. That may be a leftover from old revisions.

Line 431 (new) Converts to the array to string that is further used for `foreach` iteration.

**Comment by izharaazmi** [comment]
We (including me) should make a habit to look for any pending PR before duplicating efforts, as much as possible.

**Comment by grhcj** [comment]
@izharaazmi OK, you're right, so I will wait for @zero-24's correction for this and then review again.

File: components/com_content/models/articles.php (hunk 1 above)
Comment (zero-24) [inline components/com_content/models/articles.php]: So the easiest would be to remove the line 431 i guess as there is no reason for moving the array to a string?

Comment (izharaazmi) [inline components/com_content/models/articles.php]: Easiest is to remove the line 431. This is enough regarding this PR.

However, I'd rather replace the lines 431-437 with

```php
$authorAlias = $db->quote($authorAlias);
```

Also the `if` in the line 440 is superfluous when we already have `if (!empty($first))` at line 429, and the other way around. I'd choose to remove the  `if (!empty($first))` usage.

Comment (zero-24) [inline components/com_content/models/articles.php]: it is out now with: https://github.com/zero-24/joomla-cms/commit/0822b8204616df9ba9f77778a874f5351030f928 Thanks.

**Comment by grhcj** [comment]
I have tested this item :white_check_mark: successfully on 0822b8204616df9ba9f77778a874f5351030f928<br /><br />On Review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10922">issues.joomla.org/joomla-cms/10922</a>.</sub>

**Comment by bertmert** [comment]
I have tested this item :white_check_mark: successfully on 0822b8204616df9ba9f77778a874f5351030f928<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10922">issues.joomla.org/joomla-cms/10922</a>.</sub>

**Comment by zero-24** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10922">issues.joomla.org/joomla-cms/10922</a>.</sub>

**Pull request merged**